Solutions/CohesitySecurity/Playbooks/CreateSNOWRecord/CreateSNOWRecord.json

```diff
@@ -0,0 +1,215 @@
+{
```

Comment:
Remove this playbook. Again for reference purpose only.

Reply:
removed.